tests/foreman/destructive/test_installer.py

```diff
@@ -63,8 +63,8 @@ def test_installer_sat_pub_directory_accessibility(target_sat):
     http_curl_command = f'curl -i {target_sat.url.replace("https", "http")}/pub/ -k'
     https_curl_command = f'curl -i {target_sat.url}/pub/ -k'
     for command in [http_curl_command, https_curl_command]:
-        accessibility_check = target_sat.execute(command)
-        assert 'HTTP/1.1 200 OK' or 'HTTP/2 200 ' in accessibility_check.stdout.split('\r\n')
+        accessibility_check = target_sat.execute(command).stdout.split('\r\n')
+        assert 'HTTP/1.1 200 OK' in accessibility_check or 'HTTP/2 200 ' in accessibility_check
```

Comment:
If you're just checking for specific substrings, you likely don't need to bother splitting the output into a list.
```suggestion
        accessibility_check = target_sat.execute(command).stdout
        assert 'HTTP/1.1 200 OK' in accessibility_check or 'HTTP/2 200 ' in accessibility_check
```